### DISCUSSION TITLE
How do I do real time streaming of data in loopback 4?

### DISCUSSION THREAD

**Discussion by demiro** [loopbackio/loopback-next#9336] [Q&A]
Previously in loopback 3 there was a "fairly simple" way of streaming ... I can find it in [documentation](https://loopback.io/doc/en/lb3/Realtime-server-sent-events.html) ... the thing is I have no idea on how to do this in loopback 4? No documentation provided.

**Comment by psomby** [comment]
Have a look at https://www.npmjs.com/package/@loopback/socketio or use your own implementation of socket.io anyway its bit more complicated then with LB3
